olp-cpp-sdk-core/include/olp/core/http/NetworkTypes.h

```diff
@@ -130,9 +130,14 @@ class SendOutcome final {
 CORE_API std::string ErrorCodeToString(ErrorCode code);
 
 /**
- * @brief Type alias for HTTP headers.
+ * @brief Type alias for HTTP header.
  */
-using HeadersType = std::vector<std::pair<std::string, std::string>>;
+using HeaderPair = std::pair<std::string, std::string>;
```

Comment:
```suggestion
using HeaderType = std::pair<std::string, std::string>;
```

Reply:
Renamed to Header and Headers. It is harder for humans distinguish differences in one letter inside word, than at the ends.